.github/infrastructure/docker-compose-redis7.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
+services:
+  redis:
+    image: redis:7.0.5
```

Comment:
Note there is no available redisjson image for Redis 7, so consequently Query API cannot be supported with Redis 7 at this time (it may be possible, but requires manual set up of the server)

Reply:
SHould this be just redis:7 however? So it gets upgraded to say 7.0.6, 7.1.0...